src/jingle_sip/mod_jingle_sip.erl

```diff
@@ -71,7 +82,6 @@ maybe_add_udp_max_size(NkSipOpts, Opts) ->
 -spec stop(ejabberd:server()) -> ok.
 stop(Host) ->
     ejabberd_hooks:delete(hooks(Host)),
-    nksip:stop(?SERVICE),
```

Comment:
Maybe part of Jingle/SIP extension could be extracted into `service_...`? Not stopping `nksip` seems a bit like a temporary solution.

Reply:
1. I removed the stop here since there may be many xmpp hosts served by MongooseIM. This service is shared between them so stopping it with first host stopped is bad idea, it's better to let it operate on stop manually if needed.
2. I tried `service_` framework, I was not able to use it without modifications. Below is what I miss or makes things more difficult:
     1. currently if I want to have a service started for a module, it needs to be explicitly added in the config file in the `services` section. This complicates modules configuration since the config is split into 2 places - the service and the module. I think it be more useful to allow configuring the service via module's options.
    2. even I start nksip as a service, there is still no mechanism which will stop it when all modules using it are stopped.

Both items from point 2 are not very small changes, that's why I decided not to go with the `service` framework. I think nksip service is perfect fit for the `service` framework when it supports what's missing for me at this moment.

Reply:
OK, understood, I agree it's not a big deal to let it running. Maybe some yet another alternative solution will emerge over time.